### PR TITLE
Fix broken CRAN check - fixed documentation of h2o.grep

### PR DESCRIPTION
**Pull request by michalkurka**

### CODE REVIEW

File: h2o-r/h2o-package/R/frame.R [hunk 1]
```diff
@@ -3794,7 +3794,7 @@ h2o.tolower <- function(x) .newExpr("tolower", x)
 h2o.toupper <- function(x) .newExpr("toupper", x)
 
 #'
-#' Searches for matches to argument ‘pattern’ within each element
+#' Searches for matches to argument `pattern` within each element
 #  of a string column.
```
Comment (navdeep-G) [inline h2o-r/h2o-package/R/frame.R]: Need to replace `#` with `#'`

File: h2o-r/h2o-package/R/frame.R [hunk 2]
```diff
@@ -3807,6 +3807,7 @@ h2o.toupper <- function(x) .newExpr("toupper", x)
 #' @param x An H2O frame that wraps a single string column.
 #' @param ignore.case If `TRUE` case is ignored during matching.
```
Comment (navdeep-G) [inline h2o-r/h2o-package/R/frame.R]: Don't really need ticks around `TRUE`.

**Comment by michalkurka** [comment]
@h2o-ops please test

**Comment by michalkurka** [comment]
@h2o-ops please test

**Comment by navdeep-G** [comment]
Why are pyunits running? cc @abal5

**Comment by michalkurka** [comment]
commit 6872bcfac986fce3955a7225f943c678783a24e6 tested here: http://172.17.2.151:8080/view/Branch-PR_Test/job/branch_R_cmd_check_as_Cran/88/consoleFull

Merging.

**Pull request merged**